messages/login/errors.go

```diff
@@ -0,0 +1,5 @@
+package login
+
+const (
+	ErrorCreate = "Failed to create the Personal Token: %s. Check your settings and try again. If the error persists, contact Azion support."
```

Comment:
Maybe: "Failed to login" ?

Reply:
For the end user it would be more interesting even if his login failed, if he wants more it's log level debugging